### DISCUSSION TITLE
Favorites

### DISCUSSION THREAD

**Discussion by janstehno** [janstehno/cotw-companion#36] [Ideas]
You could add some (maybe 3 or 5) from each of reserves, animals, etc. to the favorites. It would be easier and quicker to access these after.

[original discussion](https://github.com/janstehno/cotwcompanion/discussions/34)

**Comment by mercster** [comment]
Yes, like a cheat sheet feature!  Select which DB searches and lookups to include.  As a new player, I am constantly looking at the same few pages, but they are kinda awkward to keep going back and forth between.
